app/interactors/update_project/prepare_params.rb

```diff
@@ -0,0 +1,25 @@
+class UpdateProject
+  class PrepareParams
+    include Interactor
+
+    delegate :current_user, :project_params, :project, to: :context
+
+    def call
+      context.prepared_project_params = prepared_project_params
+    end
+
+    private
+
+    def prepared_project_params
+      project_params.merge(
+        {
+          user_ids: user_ids
+        }
+      )
+    end
+
+    def user_ids
+      project_params[:user_ids].push(current_user.id.to_s).uniq
```

Comment:
As I think we don`t need to add current_user id in project when we update project